docs/requirements.txt

```diff
@@ -1,27 +1,7 @@
-# If you change this file, run "make docs-requirements"
-# >=1.24.2 from kubernetes-asyncio (from API requirements), <1.25 from kfp
-urllib3>=1.24.2, <1.25
-GitPython>=2.1.0
-aiohttp>=3.5.0
-boto3>=1.9
-click==7.0
-# this is the pipelines version running in 2.8/2.10 iguazio system, for now locking it to this version
-kfp==0.2.5
-nest-asyncio>=1.0.0
 # <0.16 from botocore inside nuclio-jupyter, >=0.13.1 from readme-renderer inside twine (from dev-requirements)
 docutils<0.16, >=0.13.1
-nuclio-jupyter>=0.8.3
-pandas>=1.0.1
-pyarrow>=0.13
-pyyaml>=5.1.0
-requests>=2.20.1
-sqlalchemy>=1.3.0
-tabulate>=0.8.0, <=0.8.3
-v3io>=0.3.3
-matplotlib
-scikit-learn
-seaborn
+numpydoc==0.9.1
+recommonmark==0.6.0
+sphinx==2.2.2
+sphinx_rtd_theme==0.4.3
 sphinx-copybutton
```

Comment:
If we're already touching this - let's change the requirements to be bounded and not pinned - e.g.
`numpydoc~=0.9` instead of `numpydoc==0.9.1`